Stopped working with v2.0.0

I'm running Atom v1.5.3 & after installing v2.0.0 jshint stopped working.

@Arcanemagus `apm test` fails in for both `1.3.1` and `2`; I guess I need some package for atom development.

```sh
EChavez-MBP:linter-jshint erick.ruizdechavez$ apm test
[15786:0216/140906:ERROR:renderer_main.cc(200)] Running without renderer sandbox
[15782:0216/140907:INFO:CONSOLE(85)] "Error: Cannot find module '/Users/erick.ruizdechavez/.atom/packages/linter-jshint/spec'
  at Module._resolveFilename (module.js:336:15)
  at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:286:25)
  at Module.require (module.js:365:17)
  at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
  at requireSpecs (/Applications/Atom.app/Contents/Resources/app.asar/spec/jasmine-test-runner.js:101:7)
  at module.exports (/Applications/Atom.app/Contents/Resources/app.asar/spec/jasmine-test-runner.js:49:7)
  at module.exports (/Applications/Atom.app/Contents/Resources/app.asar/src/initialize-test-window.js:71:17)
  at setupWindow (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:86:5)
  at window.onload (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:41:9)
", source: /Applications/Atom.app/Contents/Resources/app.asar/src/initialize-test-window.js (85)
Tests failed
```

--- a/lib/main.js
+++ b/lib/main.js
@@ -101,7 +101,9 @@ module.exports = {
             return results
           }
         }
-        if (this.lintInlineJavaScript) {
+        if (this.lintInlineJavaScript &&
+          textEditor.getGrammar().scopeName.indexOf('text.html') !== -1
+        ) {
           parameters.push('--extract', 'always')
         }
         parameters.push('-')

